### DISCUSSION TITLE
AVIF and JPEG XL Support

### DISCUSSION THREAD

**Discussion by donatopace** [d2phap/ImageGlass#1547] [General]
**AVIF**
Offline method to convert:
https://avif.io/

**JPEG XL vs AVIF** (JPEG XL Win!)
https://avif.io/blog/comparisons/avif-vs-jpegxl/#qualityandlimits

JPEG XL and AVIF will be future also WebP less!

Can you add AVIF and JPEG XL to IMAGEGLASS with custom parameters of quality?

Thanks.

**Comment by d2phap** [comment]
ImageGlass already supports viewing AVIF and JXL. You can convert image to JXL as well.

**Reply by d2phap** [reply]
for the quality, you can change it in Settings > Edit > Image quality

![image](https://user-images.githubusercontent.com/3154213/232384803-feb28f1a-a320-4ee6-aeb2-08a207d423d9.png)

**Reply by donatopace** [reply]
OK,
in the other discussion we have spoken also of
"Thumbnail" Support:
https://github.com/d2phap/ImageGlass/discussions/1562
